frontend/pulse3d/src/pages/uploads.js

```diff
@@ -247,7 +249,7 @@ export default function Uploads() {
             checked: isChecked,
           };
         });
-        setJobs(newJobs);
+        setJobs([...newJobs]);
```

Comment:
Why is this change necessary? Just curious

Reply:
Oh no.. I think i did this when I was trying to see why the loading spinner never disappeared and why the table was rerendering so often

Reply:
good catch, I'll swap it back. It doesn't matter